admin/src/components/Menu/index.js

```diff
@@ -0,0 +1,14 @@
+import { h, Component } from 'preact';
+import { Link } from 'preact-router/match';
+
+export default class Menu extends Component {
+  render() {
+    return (
+      <nav>
+        <Link href="/">All comments</Link>
```

Comment:
it's not really possible to do a proper link here, as the files are served as static content. So we'll need to think about this.

Reply:
Those links redirects to components defined in router. Everything is good here